xml/troubleshooting.xml

```diff
@@ -1079,64 +1079,67 @@
+   <title>&gnome; Desktop Has Issues</title>
    <para>
-    If this is the case, it is likely that your &gnome; configuration files
-    have become corrupted. Some symptoms may include the keyboard failing to
-    work, the screen geometry becoming distorted, or even the screen coming up
-    as a bare gray field. The important distinction is that if another user
-    logs in, the machine works normally. It is then likely that the problem can
-    be fixed relatively quickly by simply moving the user's &gnome;
-    configuration directory to a new location, which causes &gnome; to
-    initialize a new one. Although the user is forced to reconfigure &gnome;,
-    no data is lost.
+     
+     If you are experiencing issues with the &gnome; desktop (for
+     example, incorrect screen geometry, blank desktop, etc.), there
+     are several ways to troubleshoot the misbehaving graphical
+     desktop environment. The recommended procedure described below
+     offers the safest option to fix broken &gnome;.
    </para>
-   <procedure xml:id="proc-trouble-login-dk">
-    <step>
-     <para>
-      Switch to a text console by pressing <keycombo>
-      <keycap function="control"/> <keycap function="alt"/> <keycap>F1</keycap>
-      </keycombo>.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in with your user name.
-     </para>
-    </step>
-    <step>
-     <para>
-      Move the user's &gnome; configuration directories to a temporary
-      location:
-     </para>
-<screen>&prompt.user;mv .gconf  .gconf-ORIG-RECOVER
-&prompt.user;mv .gnome2 .gnome2-ORIG-RECOVER</screen>
-    </step>
-    <step>
-     <para>
-      Log out.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in again, but do not run any applications.
-     </para>
-    </step>
-    <step>
-     <para>
-      Recover your individual application configuration data (including the
-      Evolution e-mail client data) by copying the
-      <filename>~/.gconf-ORIG-RECOVER/apps/</filename> directory back into the
-      new <filename>~/.gconf </filename> directory as follows:
-     </para>
-<screen>&prompt.user;cp -a .gconf-ORIG-RECOVER/apps .gconf/</screen>
-     <para>
-      If this causes the login problems, attempt to recover only the critical
-      application data and reconfigure the remainder of the applications.
-     </para>
-    </step>
+   <procedure>
+     <title>Troubleshooting &gnome;</title>
+     <step>
+       <para>
+	 Launch &yast; and switch to <guimenu>Security and Users</guimenu>.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Open the <guimenu>User and Group Management</guimenu> dialog and press <guimenu>Add</guimenu>.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Fill out the required fields and press <guimenu>OK</guimenu> to create a new user.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Log out and log in as the new user. This gives with a fresh
+	 installation of &gnome;.
```

Comment:
More precisely, instead of providing a "fresh installation", log-in as a new user provides "a fresh GNOME user environment" or "a fresh user GNOME session".